src/main/java/net/imglib2/algorithm/edge/SubpixelEdgelDetection.java

```diff
@@ -73,7 +73,8 @@ public class SubpixelEdgelDetection
 	 * <p>
 	 * Note: The input image type must be a signed type! Otherwise gradient
 	 * computation will not work.
-	 * 
+	 *
+	 * @implNote op name='image.subpixelEdgels', type=Function
```

Comment:
This maybe belongs better in the `features` namespace?

Reply:
Sure, rename it.